streaming/src/main/scala/org/apache/spark/streaming/dstream/FileInputDStream.scala

```diff
@@ -253,13 +254,19 @@ class FileInputDStream[K, V, F <: NewInputFormat[K,V]](
   private def filesToRDD(files: Seq[String]): RDD[(K, V)] = {
     val fileRDDs = files.map(file =>{
       val rdd = serializableConfOpt.map(_.value) match {
-        case Some(config) => context.sparkContext.newAPIHadoopFile(
+        case Some(config) => {
+          if(flag){ println("PROCESSING FILE :"+file) }
```

Comment:
I think the reason why I file a wish JIRA to expose file name is that user want to get the file name in the run-time, but here your implementation is just println the file name on the console, so for end-user, how to use this file name? I think your current implementation cannot well address the problem.